doc/operations/start_network.rst

```diff
@@ -13,41 +13,23 @@ To create a new CCF network, the first node of the network should be started wit
 
 .. code-block:: bash
 
-    $ cchost
-    --enclave-file /path/to/enclave_library
-    --rpc-address <ccf-node-address>
-    --node-address <ccf-node-to-node-address>
-    --public-rpc-address <ccf-node-public-address>
-    --ledger-dir /path/to/ledger/dir
-    --node-cert-file /path/to/node_certificate
-    start
-    --network-cert-file /path/to/network_certificate
-    --member-info /path/to/member1_cert[,/path/to/member1_enc_pubk[,/path/to/member1_data]]
-    [--member-info /path/to/member2_cert[,/path/to/member2_enc_pubk[,/path/to/member2_data]] ...]
-    --constitution /path/to/javascript/constitution_module.js
+    $ cchost --config /path/to/config/file start
 
 .. mermaid::
 
     graph LR;
         Uninitialized-- config -->Initialized;
         Initialized-- start -->PartOfNetwork;
 
-CCF nodes can be started by using IP Addresses (both IPv4 and IPv6 are supported) or by specifying a fully qualified domain name. If an FQDN is used then ``--san dNSName:<sub.domain.tld>`` should be passed to the node at startup. Once a DNS has been setup it will be possible to connect to the node over TLS by using the node's domain name.
-
-When starting up, the node generates its own key pair and outputs the unendorsed certificate associated with its public key at the location specified by ``--node-cert-file``. The certificate of the freshly-created CCF network is also output at the location specified by ``--network-cert-file``.
-
 The unique identifier of a CCF node is the hex-encoded string of the SHA-256 digest the public key contained in its identity certificate (e.g. ``50211327a77fc16dd2fba8fae5fffac3df909fceeb307cf804a4125ae2679007``). This unique identifier should be used by operators and members to refer to this node with CCF (for example, when :ref:`governance/common_member_operations:Trusting a New Node`).
 
-.. note:: The network certificate should be distributed to users and members to be used as the certificate authority (CA) when establishing a TLS connection with any of the nodes part of the CCF network. When using ``curl``, this is passed as the ``--cacert`` argument.
+CCF nodes can be started by using IP Addresses (both IPv4 and IPv6 are supported) or by specifying a fully qualified domain name. If an FQDN is used then a ``dNSName`` subject alternative name should be specified as part of the ``node_certificate.subject_alt_names`` configuration entry. Once a DNS has been setup it will be possible to connect to the node over TLS by using the node's domain name.
```

Comment:
We ought to have one end to end test that uses IPv6 if we make this explicit claim.

Reply:
See #3215